### PR TITLE
fix(shouldPreventDefaultOnKeyDown): return false for events that are …

### PR DESCRIPTION
**Pull request by ling1726**
Repro: https://codesandbox.io/s/intelligent-sky-vp09ns?file=/example.tsx

Each click of the trigger menu item will call `onClick` twice

## Current Behavior
`shouldPreventDefaultOnKeyDown` still runs for default prevented events

## New Behavior

`shouldPreventDefaultOnKeyDown` returns false immediately for default prevented events

## Related Issue(s)

Fixes #

### CODE REVIEW

File: packages/react-utilities/src/utils/shouldPreventDefaultOnKeyDown.ts [hunk 1]
```diff
@@ -7,6 +7,11 @@ import * as React from 'react';
  * Useful for situations where a keydown needs to be transformed to a click event
  */
 export function shouldPreventDefaultOnKeyDown(e: KeyboardEvent | React.KeyboardEvent) {
+  console.log(e.defaultPrevented);
```
Comment (ling1726) [inline packages/react-utilities/src/utils/shouldPreventDefaultOnKeyDown.ts:10]:
```suggestion
```

Comment (layershifter) [inline packages/react-utilities/src/utils/shouldPreventDefaultOnKeyDown.ts]:
```suggestion
```

No thanks 👍

Comment (ling1726) [reply packages/react-utilities/src/utils/shouldPreventDefaultOnKeyDown.ts]: yeah removed already just before this comment 😋

**Comment by fabricteam** [comment]
## 📊 Bundle size report

| Package & Exports                                                                                                                                                             |      Baseline (minified/GZIP) |                            PR |                                                                                                                                                                                                                                               Change |
| :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: | ----------------------------: | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `171.678 kB`<br />`48.075 kB` | `171.709 kB`<br />`48.089 kB` | `31 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /><br />`14 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /> |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                                                          | `103.399 kB`<br />`31.844 kB` |  `103.43 kB`<br />`31.858 kB` | `31 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /><br />`14 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /> |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                                             | `105.754 kB`<br />`32.205 kB` | `105.785 kB`<br />`32.219 kB` | `31 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /><br />`14 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /> |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr>                                                                                 |  `96.308 kB`<br />`29.387 kB` |  `96.339 kB`<br />`29.399 kB` | `31 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /><br />`12 B` <img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" /> |

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                  |         Size (minified/GZIP) |
| -------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------: |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr>                | `72.822 kB`<br />`22.054 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                         | `44.943 kB`<br />`13.043 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>                                                            |  `20.869 kB`<br />`6.549 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr>                                              |  `21.737 kB`<br />`6.843 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr>                                            |  `21.827 kB`<br />`6.519 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr>                                                         |  `27.982 kB`<br />`8.043 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr>                                         |  `33.246 kB`<br />`9.007 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr>                                                 |   `29.65 kB`<br />`8.624 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr>                                               |  `36.132 kB`<br />`9.835 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr>                                             |   `37.28 kB`<br />`8.647 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                     |  `53.205 kB`<br />`15.27 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                               | `48.898 kB`<br />`14.083 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                   |   `7.653 kB`<br />`3.246 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                   |   `8.931 kB`<br />`3.689 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                 |   `7.626 kB`<br />`3.272 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox</abbr>                                                   |   `6.813 kB`<br />`2.895 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr> | `32.526 kB`<br />`10.645 kB` |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr>                                                      |  `15.301 kB`<br />`5.532 kB` |
| <samp>react-image</samp> <br /> <abbr title='bundle-size/Image.fixture.js'>Image</abbr>                                                            |  `10.105 kB`<br />`3.952 kB` |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/Input.fixture.js'>Input</abbr>                                                            |  `21.538 kB`<br />`7.134 kB` |
| <samp>react-label</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Label</abbr>                                                            |   `8.341 kB`<br />`3.487 kB` |
| <samp>react-link</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Link</abbr>                                                              |  `11.102 kB`<br />`4.504 kB` |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr>                                                         |   `6.267 kB`<br />`2.168 kB` |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePopper</abbr>                                              |   `23.21 kB`<br />`8.084 kB` |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr>                                       |   `14.009 kB`<br />`5.25 kB` |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/Select.fixture.js'>Select</abbr>                                                         |   `7.754 kB`<br />`3.258 kB` |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/Slider.fixture.js'>Slider</abbr>                                                         |  `22.975 kB`<br />`7.769 kB` |
| <samp>react-spinner</samp> <br /> <abbr title='bundle-size/Spinner.fixture.js'>Spinner</abbr>                                                      |   `6.811 kB`<br />`2.895 kB` |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/Switch.fixture.js'>Switch</abbr>                                                         |  `22.598 kB`<br />`7.642 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Default.fixture.js'>Text - Default</abbr>                                             |   `10.793 kB`<br />`4.23 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Wrappers.fixture.js'>Text - Wrappers</abbr>                                           |  `14.107 kB`<br />`4.573 kB` |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                                      |  `42.76 kB`<br />`14.701 kB` |
| <samp>react-utilities</samp> <br /> <abbr title='bundle-size/SSRProvider.fixture.js'>SSRProvider</abbr>                                            |         `189 B`<br />`161 B` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/9955166acd95a1328ed09efd177e9f08eeabee57'>9955166acd95a1328ed09efd177e9f08eeabee57</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Avatar-mount_base.html">769</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Avatar-mount.html">784</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/BaseButton-mount_base.html">784</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/BaseButton-mount.html">825</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Breadcrumb-mount_base.html">2302</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Breadcrumb-mount.html">2317</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ButtonNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/ButtonNext-mount_base.html">414</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/ButtonNext-mount.html">437</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Checkbox-mount_base.html">1348</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Checkbox-mount.html">1359</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/CheckboxBase-mount_base.html">1165</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/CheckboxBase-mount.html">1141</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/ChoiceGroup-mount_base.html">4162</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/ChoiceGroup-mount.html">4125</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/ComboBox-mount_base.html">859</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/ComboBox-mount.html">857</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/CommandBar-mount_base.html">8914</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/CommandBar-mount.html">8921</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/ContextualMenu-mount_base.html">7417</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/ContextualMenu-mount.html">7427</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/DefaultButton-mount_base.html">984</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/DefaultButton-mount.html">999</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/DetailsRow-mount_base.html">3276</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/DetailsRow-mount.html">3246</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/DetailsRowFast-mount_base.html">3247</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/DetailsRowFast-mount.html">3217</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/DetailsRowNoStyles-mount_base.html">3111</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/DetailsRowNoStyles-mount.html">3097</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Dialog-mount_base.html">1952</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Dialog-mount.html">1952</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/DocumentCardTitle-mount_base.html">155</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/DocumentCardTitle-mount.html">171</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Dropdown-mount_base.html">2778</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Dropdown-mount.html">2783</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/FluentProviderNext-mount_base.html">1685</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/FluentProviderNext-mount.html">1686</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/FluentProviderWithTheme-mount_base.html">178</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/FluentProviderWithTheme-mount.html">166</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/FluentProviderWithTheme-virtual-rerender_base.html">101</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/FluentProviderWithTheme-virtual-rerender.html">94</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">175</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount.html">151</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/FocusTrapZone-mount_base.html">1597</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/FocusTrapZone-mount.html">1612</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/FocusZone-mount_base.html">1534</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/FocusZone-mount.html">1597</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/IconButton-mount_base.html">1527</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/IconButton-mount.html">1508</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Label-mount_base.html">317</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Label-mount.html">320</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Layer-mount_base.html">2590</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Layer-mount.html">2558</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Link-mount_base.html">467</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Link-mount.html">434</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/MakeStyles-mount_base.html">1477</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/MakeStyles-mount.html">1472</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/MenuButton-mount_base.html">1279</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/MenuButton-mount.html">1292</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/MessageBar-mount_base.html">1758</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/MessageBar-mount.html">1759</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Nav-mount_base.html">2820</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Nav-mount.html">2855</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/OverflowSet-mount_base.html">992</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/OverflowSet-mount.html">991</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Panel-mount_base.html">1897</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Panel-mount.html">1906</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Persona-mount_base.html">751</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Persona-mount.html">742</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Pivot-mount_base.html">1263</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Pivot-mount.html">1279</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/PrimaryButton-mount_base.html">1139</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/PrimaryButton-mount.html">1145</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Rating-mount_base.html">6727</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Rating-mount.html">6706</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/SearchBox-mount_base.html">1200</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/SearchBox-mount.html">1176</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Shimmer-mount_base.html">2214</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Shimmer-mount.html">2213</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Slider-mount_base.html">1740</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Slider-mount.html">1699</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/SpinButton-mount_base.html">4312</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/SpinButton-mount.html">4334</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Spinner-mount_base.html">400</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Spinner-mount.html">390</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/SplitButton-mount_base.html">2750</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/SplitButton-mount.html">2715</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Stack-mount_base.html">477</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Stack-mount.html">474</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/StackWithIntrinsicChildren-mount_base.html">2012</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/StackWithIntrinsicChildren-mount.html">2023</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/StackWithTextChildren-mount_base.html">4567</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/StackWithTextChildren-mount.html">4592</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/SwatchColorPicker-mount_base.html">9982</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/SwatchColorPicker-mount.html">10024</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/TagPicker-mount_base.html">2285</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/TagPicker-mount.html">2249</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/TeachingBubble-mount_base.html">11490</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/TeachingBubble-mount.html">11475</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Text-mount_base.html">385</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Text-mount.html">409</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/TextField-mount_base.html">1227</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/TextField-mount.html">1223</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/ThemeProvider-mount_base.html">1046</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/ThemeProvider-mount.html">1053</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/ThemeProvider-virtual-rerender_base.html">550</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/ThemeProvider-virtual-rerender.html">550</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1617</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1616</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Toggle-mount_base.html">713</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/Toggle-mount.html">735</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/buttonNative-mount_base.html">135</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/21905/perf-test/buttonNative-mount.html">130</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Pull request merged**